lib/src/components/media-card.ts

```diff
@@ -0,0 +1,80 @@
+import '../globals';
+
+const { LitElement, html, css } = window.Lit;
+
+class MediaCard extends LitElement {
+    static styles = css`
+        
+        .wrapper {
+            display: flex;
+            justify-content: center;
+            flex-wrap: wrap;
+            margin-top: 6.25rem;
+        }
+
+        .container {
+            background: white;
+            width: 19.18rem;
```

Comment:
Standardize this, either by rounding up to 20rem units, OR by using `aspect-ratio` CSS ruleset.

Reply:
Standardized to 20rem.